geometry_msgs/Point point_tl
geometry_msgs/Point point_tr
geometry_msgs/Point point_bl

bool exists_roi